### PR TITLE
Update macOS export page to reflect rcodesign support.

### PR DESCRIPTION
**Pull request by bruvzg**
Update macOS export page to reflect changes from https://github.com/godotengine/godot/pull/64207 (rcodesign support, some options are selected automatically and removed for the export config).

### CODE REVIEW

File: tutorials/export/exporting_for_macos.rst [hunk 1]
```diff
@@ -31,25 +29,58 @@ By default, macOS will run only applications that are signed and notarized. If y
...
 
-If you have an Apple Developer ID Certificate
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you have an Apple Developer ID Certificate and exporting from macOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Export your project from a computer running macOS with Xcode command line tools installed.
-- Enable ``Code Signing``, ``Notarization``, ``Hardened Runtime`` and ``Timestamp`` and disable the ``Debug`` entitlement.
-- Provide valid Apple ID credentials and certificate identity.
+Install `Xcode <https://developer.apple.com/xcode/>`__ command line tools and open Xcode at least once or run ``sudo xcodebuild -license accept`` command to accept license agreement.
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_macos.rst:35]:
```suggestion
Install `Xcode <https://developer.apple.com/xcode/>`__ command line tools and open Xcode at least once or run the ``sudo xcodebuild -license accept`` command to accept license agreement.
```

File: tutorials/export/exporting_for_macos.rst [hunk 2]
```diff
@@ -31,25 +29,58 @@ By default, macOS will run only applications that are signed and notarized. If y
...
 You can use the ``xcrun notarytool history`` command to check notarization status and use the ``xcrun notarytool log {ID}`` command to download the notarization log.
 
 If you encounter notarization issues, see `Resolving common notarization issues <https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/resolving_common_notarization_issues>`__ for more info.
 
 After notarization is completed, `staple the ticket <https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/customizing_the_notarization_workflow>`__ to the exported project.
 
+If you have an Apple Developer ID Certificate and exporting from Linux or Windows
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install `PyOxidizer rcodesign <https://github.com/indygreg/PyOxidizer/tree/main/apple-codesign>`__, and configure path to the rcodesign in the ``Editor Settings > Export > macOS > rcodesign``.
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_macos.rst:59]:
```suggestion
Install `PyOxidizer rcodesign <https://github.com/indygreg/PyOxidizer/tree/main/apple-codesign>`__, and configure the path to ``rcodesign`` in the ``Editor Settings > Export > macOS > rcodesign``.
```

File: tutorials/export/exporting_for_macos.rst [hunk 3]
```diff
@@ -31,25 +29,58 @@ By default, macOS will run only applications that are signed and notarized. If y
...
+To notarize exported app
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Select ``PyOxidizer rcodesign`` in the ``Notarization > Notarization`` option.
+- Disable the ``Debugging`` entitlement.
+- Set valid `App Store Connect <https://developer.apple.com/documentation/appstoreconnectapi>`__ API UUID / Key in the ``Notarization`` section.
+
+You can use the ``rcodesign notary-log`` command to check notarization status.
+
+After notarization is completed, use ``rcodesign staple`` command to staple the ticket to the exported project.
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_macos.rst:76]:
```suggestion
After notarization is completed, use the ``rcodesign staple`` command to staple the ticket to the exported project.
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks for keeping this updated! 🎉 Merged.

**Comment by bruvzg** [comment]
Note: Can be cherry-picked for 3.6 after https://github.com/godotengine/godot/pull/66093 is merged.

**Pull request merged**